### PR TITLE
Updated NIST 800-63-2 References to NIST 800-63-3

### PR DESCRIPTION
**Pull request by gramidt**
NIST 800-63-2 has been superseded by NIST 800-63-3 as of June 22, 2017.

### CODE REVIEW

**Comment by amoose** [comment]
Yes! Please also remove LOA1/3 reference and replace with IAL1/2!

**Comment by gramidt** [comment]
@amoose Great call! I have updated my PR with the suggested changes.

File: _pages/attributes.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
 title: User attributes
 lead: >
-  login.gov user accounts are either proofed (LOA3) or not (LOA1), corresponding to <a href="http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-63-2.pdf">NIST 800-63-2</a> levels of assurance (LOA).
+  login.gov user accounts are either proofed (IAL2) or (IAL1), corresponding to <a href="http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-63-3.pdf">NIST 800-63-3</a> Identity Assurance Level (IAL).
```
Comment (amoose) [inline _pages/attributes.md]: please add "self-asserted" in front of "(IAL1)"

Comment (gramidt) [reply _pages/attributes.md]: Great idea! Just updated.

**Comment by gramidt** [comment]
Thanks again for all of your help, @amoose!

**Comment by gramidt** [comment]
LGTM

**Comment by gramidt** [comment]
@amoose @stevegsa @adunkman - Anything I can do to help get this merged?

**Pull request merged**